lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -445,7 +445,8 @@ protected function transformSignup($data, $current) {
    */
   protected function transformUser($data) {
     return [
-      'id' => $data->id,
+      'id' => $data->drupal_id,
```

Comment:
No its $data->northstar_id now because it was confusing me what ID's were what. See the changes i made: https://github.com/DoSomething/phoenix/pull/6324/files#diff-d0a29b0a298822b8c127c0c3ce12f46dR123